app/controlplane/pkg/data/ent/migrate/migrations/20241019145455.sql

```diff
@@ -0,0 +1,2 @@
+-- Modify "workflows" table
+ALTER TABLE "workflows" ALTER COLUMN "project" SET NOT NULL;
```

Comment:
We don't need a migration of previous workflows right?

Reply:
No because they happened to be set and forced in another layer.